Assets/Countly/Plugins/CountlySDK/Services/AbstractBaseService.cs

```diff
@@ -105,6 +124,11 @@ protected string TrimValue(string fieldName, string v)
                 v = v.Substring(0, _configuration.GetMaxValueSize());
             }
 
+            if (v == " " || v == "") {
```

Comment:
this TrimValue now has 2 jobs. lets not do that. just use the Trim() method in C# to trim the string values you receive for user properties. and just check empty space normally and print warning.

Reply:
updated